docs/docs/articles/webhooks.mdx

```diff
@@ -268,6 +279,92 @@ They can be triggered by the following resources:
+- `TestType` - Test type (for example, `postman/collection`)
+- `Name` -  Execution name (for example, `postman-executor-smoke-937)
+- `Number` - Execution number (for example, `937`)
+- `Envs` -  List of ENV variables for specific Test (if defined)
+- `Command` - Command executed inside the Pod (for example, `newman`)
+- `Args` -  Command arguments (for example, `run <runPath> -e <envFile> --reporters cli,json --reporter-json-export <reportFile>`)
+- `Variables` - List of variables
+- `Content` - Test content
+- `StartTime` - Test start time (for example, `2023-09-06 19:23:34.543433547 +0000 UTC`)
+- `EndTime` - Time when the test execution finished (for example, `2023-09-06 19:23:42.221493031 +0000 UTC`)
+- `Duration` - Test duration in seconds (for example, `7.68s`)
+- `DurationMs` - Test duration in miliseconds (for example, `7678`)
+- `ExecutionResult` - Execution result (https://github.com/kubeshop/testkube/blob/main/pkg/api/v1/testkube/model_event.go)
+- `Labels` Test labels (for example, `[core-tests:executors executor:postman-executor test-type:postman-collection],`)
+- `RunningContext` - Running context - how the test has been triggered (for example, `user-ui`)
+
+Full Execution data model can be found [here](https://github.com/kubeshop/testkube/blob/main/pkg/api/v1/testkube/model_execution.go)
+
+### TestSuiteExecution:
+
+- `Id` - TestSuiteExecution ID (for example, `64f8d5b2712890925aea51dc`)
+- `Name` - TestSuite name (for example, `ts-executor-postman-smoke-tests-472`)
+- `Status` - TestSuite execution status (for example, `running` or `passed`)
+- `Envs` - List of ENV variables
+- `Variables` - List of variables
+- `StartTime` - Test start time (for example, `2023-09-06 19:23:34.543433547 +0000 UTC`)
+- `EndTime` - Time when the test execution finished (for example, `2023-09-06 19:23:42.221493031 +0000 UTC`)
+- `Duration` - Test duration in seconds (for example, `7.68s`)
+- `DurationMs` - Test duration in miliseconds (for example, `7678`)
+- `StepResults`
+- `Labels` - TestSuite labels (for example, `[app:testkube testsuite:executor-postman-smoke-tests]`)
+- `RunningContext` - Running context - how the TestSuite has been triggered (for example, `user-ui`)
+
+Full TestSuiteExecution data model can be found [here](https://github.com/kubeshop/testkube/blob/main/pkg/api/v1/testkube/model_test_suite_execution.go)
+
+## Additional examples
+
+### Microsoft Teams
+Webhooks can also be used to send messages to Microsoft Teams channels.
+First, you need to create an incoming webhook at Teams for specific channel. You can check how to do it at Teams Docs [here](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=dotnet#create-incoming-webhooks-1). After your Teams incoming webhook is created, you can use it with Testkube webhooks - just use the URL provided (will probably look like this: `https://xxxxx.webhook.office.com/xxxxxxxxx`).
```

Comment:
```suggestion
First, you need to create an incoming webhook in Teams for a specific channel. You can see how to do it in the Teams Docs [here](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=dotnet#create-incoming-webhooks-1). After your Teams incoming webhook is created, you can use it with Testkube webhooks - just use the URL provided (it will probably look like this: `https://xxxxx.webhook.office.com/xxxxxxxxx`).
```